corehq/apps/hqwebapp/static/hqwebapp/js/alpinejs/directives/datepicker.js

```diff
@@ -1,6 +1,37 @@
 import { TempusDominus } from '@eonasdan/tempus-dominus';
 import '@eonasdan/tempus-dominus/dist/css/tempus-dominus.css';
 import Alpine from 'alpinejs';
+import _ from 'underscore';
+
+const defaultTranslations = {
```

Comment:
Can you move these into a constants file that can be shared with `tempus_dominus.js`?

Reply:
great idea, adding to todos